source/tyr/migrations/versions/44dab62dfff7_add_shape_on_user.py

```diff
@@ -0,0 +1,22 @@
+"""empty message
```

Comment:
You should explain what this migration do, this text will be shown at the execution of the migration